neo/app/controllers/Application.scala

```
@@ -72,6 +76,68 @@ class Application @Inject()(val wsClient: WSClient,
     }
   }
 
+  def loadCity() : List[JsValue] = {
+    val file = environment.getFile("/public/data/city.sample.shorten.json")
+    val stream = new FileInputStream(file)
+    val json = Json.parse(stream)
+    val values = (json \ "features").as[List[JsObject]]
+    var newValues = List[JsValue]()
+    for(n <- values.indices) {
+      val thisValue = values.apply(n)
+      val geoType = (thisValue \ "geometry" \ "type").as[String]
+      var sumX = 0.0
+      var sumY = 0.0
+      var length = 0.0
+      if(geoType == "Polygon") {
+        val rawCorr = (thisValue \ "geometry" \ "coordinates").as[JsArray]
+        val realCorr = rawCorr.apply(0).as[JsArray]
+        for(i <- realCorr.value.indices){
+          sumX += realCorr.apply(i).apply(0).as[Double]
+          sumY += realCorr.apply(i).apply(1).as[Double]
+        }
+        length = realCorr.value.size
+      } else if(geoType == "MultiPolygon") {
+        val allCorr = (thisValue \ "geometry" \ "coordinates").as[JsArray]
+        for(i <- allCorr.value.indices){
+          val rawCorr = allCorr.apply(i).as[JsArray]
+          val realCorr = rawCorr.apply(0).as[JsArray]
+          for(j <- realCorr.value.indices){
+            sumX += realCorr.apply(j).apply(0).as[Double]
+            sumY += realCorr.apply(j).apply(1).as[Double]
+          }
+          length += realCorr.value.size
+        }
+      } else {
+        throw new IllegalArgumentException("Unidentified geometry type in city.json");
```

use log and ignore the error point?